src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceControlManagerClient.cs

```diff
@@ -0,0 +1,1913 @@
+        {
+            ErrorCode error = VcMgrSetSelectedResults(list._handle);
+            if (error != ErrorCode.None)
+            {
+                Log.Error(LogTag, "SetSelectedResults Failed with error " + error);
+                throw ExceptionFactory.CreateException(error);
+            }
+        }
+
+        /// <summary>
+        /// Called when client gets the all recognition results from vc-daemon.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<SetAllResultEventArgs> SetAllResult
+        {
+            add
+            {
+                _allResultDelegate = (ResultEvent evt, IntPtr vcCmdList, IntPtr result, IntPtr msg, IntPtr userData) =>
+                {
+                    string resultString = Marshal.PtrToStringAnsi(result);
+                    string msgString = Marshal.PtrToStringAnsi(msg);
+                    VoiceCommandList cmdList = new VoiceCommandList(new SafeCommandListHandle(vcCmdList));
+                    SetAllResultEventArgs args = new SetAllResultEventArgs(evt, cmdList, resultString, msgString);
+                    _setAllResult?.Invoke(null, args);
+                    return true;
+                };
+                ErrorCode error = VcMgrSetAllResultCb(_allResultDelegate, IntPtr.Zero);
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Add SetAllResult Failed with error " + error);
+                }
+
+                else
+                {
+                    _setAllResult += value;
+                }
+            }
+
+            remove
+            {
+                ErrorCode error = VcMgrUnsetAllResultCb();
+                if (error != ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Remove SetAllResult Failed with error " + error);
+                }
+
+                _setAllResult -= value;
+                _allResultDelegate = null;
+            }
+        }
+
+        /// <summary>
+        /// Called when client gets the pre recognition results(partial ASR) from vc-daemon.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+        /// <privlevel>public</privlevel>
+        /// <feature>http://tizen.org/feature/speech.control</feature>
+        /// <feature>http://tizen.org/feature/microphone</feature>
+        /// <pre>
+        /// The State should be Initialized
+        /// </pre>
+        public static event EventHandler<PreResultEventArgs> PreRecognitionResultUpdated
```

Comment:
`PreResultEventArgs` also changed to `PreRecognitionResultUpdatedEventArgs`

Reply:
Fixed.